src/javascript/app/common/account_opening.js

```diff
@@ -155,7 +155,7 @@ const AccountOpening = (() => {
                 }
 
                 const $tax_residence_select = $('#tax_residence');
-                $tax_residence_select.html($options_with_disabled.html());
+                $tax_residence_select.html($options.html());
```

Comment:
they keep asking us to change this every few weeks 😂

Reply:
We'll see what happens I guess, haha.